### PR TITLE
[BLZT-10,11] added tests and started java doc

### PR DESCRIPTION
**Pull request by romanovosad87**

### CODE REVIEW

File: web/src/main/java/com/bobocode/bring/web/TomcatApplication.java [hunk 1]
```diff
@@ -1,7 +1,19 @@
 package com.bobocode.bring.web;
 
+/**
+ * The {@code TomcatApplication} class contains the main method to bootstrap the Bring application using Tomcat.
+ * It serves as the entry point for starting the application context and initializing the embedded Tomcat server.
+ *
+ * @author Blyzhnytsia Team
+ * @since 1.0
+ */
 public class TomcatApplication {
```
Comment (PolukovY) [inline web/src/main/java/com/bobocode/bring/web/TomcatApplication.java:10]: Need to remove later we don't need it.

File: web/src/test/java/com/bobocode/bring/web/RestControllerTest.java [hunk 2]
```diff
@@ -26,21 +28,19 @@ public class RestControllerTest {
...
     public static final String URL = "http://localhost:%s%s";
+    public static final String APPLICATION_JSON = "application/json";
     private static ObjectMapper objectMapper;
     private static ServerProperties serverProperties;
     private HttpClient httpClient;
 
     @BeforeAll
     static void beforeAll() {
-        BringApplicationContext bringApplicationContext = BringWebApplication.run("testdata");
+        BringApplicationContext bringApplicationContext = BringWebApplication.run("testdata.generalIntegrationTest");
```
Comment (PolukovY) [inline web/src/test/java/com/bobocode/bring/web/RestControllerTest.java:43]: should be lowercase testdata.generallntegrationtest

**Review by PolukovY — APPROVED** [review]

File: web/src/main/java/com/bobocode/bring/web/TomcatApplication.java (hunk 1 above)
Comment (romanovosad87) [reply web/src/main/java/com/bobocode/bring/web/TomcatApplication.java:10]: Agreed!

File: web/src/test/java/com/bobocode/bring/web/RestControllerTest.java (hunk 2 above)
Comment (romanovosad87) [reply web/src/test/java/com/bobocode/bring/web/RestControllerTest.java]: Done

**Review by PolukovY — APPROVED** [review]

**Pull request merged**